app/scripts/lib/ens-ipfs/resolver.js

```diff
@@ -1,7 +1,7 @@
 import namehash from 'eth-ens-namehash';
 import Eth from 'ethjs-query';
 import EthContract from 'ethjs-contract';
-import contentHash from 'content-hash';
+import contentHash from '@ensdomains/content-hash';
```

Comment:
Hi. Makoto from ENS team. 
FYI,`content-hash` is no longer supported by the original author so we forked and took over the maintenance, so please use '@ensdomains/content-hash' as in this PR